remixer_core/src/main/java/com/google/android/libraries/remixer/Variable.java

```diff
@@ -185,6 +190,13 @@ public void setValueWithoutNotifyingOthers(T newValue) {
   }
 
   /**
+   * Gets the serializable constraints string for this variable.
+   */
+  public String getSerializableConstraints() {
+    return SERIALIZABLE_CONSTRAINTS;
```

Comment:
makes more sense for all these constants to be defined in StoredVariable rather than separately